core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages.properties

```diff
@@ -74,7 +74,7 @@ BiasedLockingRevocationRule_CONFIG_WARNING_LIMIT_LONG=The number of revocations
 BiasedLockingRevocationRule_NAME=Biased Locking Revocation
 BiasedLockingRevocationRule_TEXT_OK=No classes were disabled from participating in biased locking
 BiasedLockingRevocationRule_TEXT_REVOKED_CLASSES_FOUND=Fully revoked classes found.
-BiasedLockingRevocationRule_TEXT_REVOKED_CLASSES_FOUND_LONG=The following classes were disabled from using biased locking: {revokedTypes}
+BiasedLockingRevocationRule_TEXT_REVOKED_CLASSES_FOUND_LONG=The following classes were disabled from using biased locking: {revokedClasses}
```

Comment:
Need to update copyright year.

Reply:
Thank you for your advice. I fixed it in 24db383.